### PR TITLE
Adding support for config toml files to parse flags

### PR DESCRIPTION
**Pull request by Rachit1313**
I would be working on issue #11 through this PR.

### CODE REVIEW

**Comment by Rachit1313** [comment]
Hi,
The implementation for the feature has been completed from my side.

Could you please review it and let me know if any changes are required?

Thank you.

File: txt2html.py [hunk 1]
```diff
@@ -205,6 +206,18 @@ def main():
...
 
+    if args.config:
+        config_file_path = args.config
+        try:
+            with open(config_file_path, "rb") as config_file:
+                config_data = tomli.load(config_file)
+
+                lang_attribute_value = config_data.get("lang", lang_attribute_value)
+        except FileNotFoundError:
+            print(f"Config file not found: {config_file_path}")
```
Comment (mingming-ma) [inline txt2html.py:217]:
```suggestion
            parser.error(f"Config file not found: {config_file_path}")
```

We need to exit here if If the file is missing, or can't be parsed as TOML

File: txt2html.py [hunk 2]
```diff
@@ -205,6 +206,18 @@ def main():
...
+        except tomli.TOMLDecodeError:
+            print(f"Error parsing the config file: {config_file_path}")
```
Comment (mingming-ma) [inline txt2html.py:219]:
```suggestion
            parser.error(f"Error parsing the config file: {config_file_path}")
```

Same reason as line 217.

**Review by mingming-ma — CHANGES_REQUESTED** [review]

File: txt2html.py [hunk 3]
```diff
@@ -205,6 +206,18 @@ def main():
     if args.lang:
         lang_attribute_value = args.lang
 
+    if args.config:
+        config_file_path = args.config
+        try:
+            with open(config_file_path, "rb") as config_file:
+                config_data = tomli.load(config_file)
+
+                lang_attribute_value = config_data.get("lang", lang_attribute_value)
```
Comment (mingming-ma) [inline txt2html.py:215]: Can you also make the config_data get the output settings?

File: README.md [hunk 4]
```diff
@@ -21,6 +22,8 @@
 - [x] Set title from input file content (version 0.1.2)
 - [x] Support Markdown file input and Italics Parsing (version 0.1.3)
 - [x] Add support for Markdown horizontal rule (`---`) to convert that to an `<hr>` tag. Support language attribute configuration (version 0.1.4)
+- [x] Add support for TOML config files to parse flags (version 0.1.4)
```
[the remark this answers is not in the document]
Comment (Rachit1313) [reply README.md:25]: sure

File: txt2html.py (hunk 1 above)
Comment (Rachit1313) [reply txt2html.py:217]: sure

File: txt2html.py (hunk 2 above)
Comment (Rachit1313) [reply txt2html.py:219]: sure

File: txt2html.py (hunk 3 above)
Comment (Rachit1313) [reply txt2html.py]: Implemented. Could you please check and confirm if anything else is required?

**Review by mingming-ma — COMMENTED** [review]
All good and approved, will do the merge it manually by the lab Step6. Thanks

**Pull request merged**